src/main/java/org/jenkinsci/plugins/displayurlapi/EnvironmentContributorImpl.java

```diff
@@ -15,9 +15,8 @@
 
     @Override
     public void buildEnvironmentFor(@NonNull Run r, @NonNull EnvVars envs, @NonNull TaskListener listener) throws IOException, InterruptedException {
-        try (DisplayURLContext ctx = DisplayURLContext.open()) {
+        try (DisplayURLContext ctx = DisplayURLContext.open(false)) {
```

Comment:
nit: maybe worth leaving the old comment around for clarity:

```suggestion
        try (DisplayURLContext ctx = DisplayURLContext.open(false)) { // environment contributor "comes from" core
```